requirements.txt

```diff
@@ -8,3 +8,5 @@ requests
 scrypt
 pyalpm
 SQLAlchemy-Continuum
+feedgen
+pytz
```

Comment:
Right now we also duplicate the required packages in the README.md, that will need adjustments as well